src/shared/components/palette/fixed-list/write-fixed-list-field.html

```diff
@@ -1,16 +1,14 @@
-<div class="form-group" [ngClass]="{'form-group-error': !fixedListControl.valid && fixedListControl.touched}">
+<div class="form-group" [ngClass]="{'form-group-error': false}">
```

Comment:
do we need `[ngClass]="{'form-group-error': false}"` at all now?

Reply:
Removed